go/exocom/src/exocom/exocom.go

```diff
@@ -30,10 +34,13 @@ type ExoCom struct {
 var upgrader = websocket.Upgrader{}
 
 // New creates a new ExoCom instance
-func New(serviceRoutes types.Routes) (*ExoCom, error) {
+func New(serviceRoutes types.Routes, hasSecurity bool) (*ExoCom, error) {
```

Comment:
I don't think hasSecutiry needs to be passed in. We could call serviceRoutes.GetHasSecurity() in this function